broker-cli/utils/index.js

```diff
@@ -13,5 +15,7 @@ module.exports = {
   loadProto,
   Big,
   handleError,
-  basicAuth
+  basicAuth,
+  grpcDeadline,
```

Comment:
do we actually want to export `grpcDeadline`, or just the interceptor?

Reply:
we could probably remove it since its not used